### PR TITLE
Miscellaneous modernization cleanup

### PR DESCRIPTION
**Pull request by NotMyFault**
This PR polishes several components:
- Utilize the computer symbol in SlaveComputer
- Use the cogwheel symbol to edit an agent, like previously.
- Adapt the color scheme on drop downs
- The text preview box is now less sharp
- Single line -> multi line expanders now expand into a jenkins-input to benefit from the modernization
- Other minor changes

### Proposed changelog entries

* Miscellaneous polishing of various components.

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [X] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change) and are in the imperative mood. [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade

### Desired reviewers

@jenkinsci/sig-ux 

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are accurate, human-readable, and in the imperative mood
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

**Review by alecharp — CHANGES_REQUESTED** [review]
I tested using a docker `jenkins/core-pr-tester` container. It seems to be fine, expect that some localization are not respected. I tested using the Locale plugin. I'm only mentioning this for global security configuration side panel addition.

File: core/src/main/resources/hudson/security/GlobalSecurityConfiguration/index.groovy
Comment (NotMyFault) [reply core/src/main/resources/hudson/security/GlobalSecurityConfiguration/index.groovy:20]: Someone would need to provide translations in the respective .properties file in the folder. I can provide preliminary translation for a couple or maybe three languages but English I feel familiar with, but I'm passing on for others.

Comment (alecharp) [reply core/src/main/resources/hudson/security/GlobalSecurityConfiguration/index.groovy:20]: I guess this is my fault and my message was misleading: I thought about the `jelly` syntax where strings are using `%` character for the i18n. If that was like this before, my apologizes and disregard my previous comment.

Comment (timja) [reply core/src/main/resources/hudson/security/GlobalSecurityConfiguration/index.groovy:20]: `_` is a method in groovy (or now an alias to 'gettext' as of recently) which looks up the resource bundle and the translation

**Review by alecharp — APPROVED** [review]

**Review by timja — APPROVED** [review]

**Comment by timja** [comment]
This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Comment by NotMyFault** [comment]
Is there anything that blocks a merge @timja?

**Comment by timja** [comment]
> Is there anything that blocks a merge @timja?

No I slipped up on not adding the label, no real rush though generally we just make sure the PRs are merged by Tuesday to make the weekly, the 24 hours is more a minimum than an actual time.

**Pull request merged**
